YLP-1326 Implement Loop Mode objects upload

CedricPerotto opened the pull request.

CedricPerotto committed:
Milliseconds durations (+raise max duration)

CedricPerotto committed:
Fix duration tests

CedricPerotto committed:
New loop mode event

CedricPerotto committed:
Changelog

CedricPerotto committed:
Replacing eventId with guid

Nas84 commented on data/types/device/alarm/alarm.go:
```diff
@@ -28,7 +28,7 @@ const (
 	Outdated                = "outdated"
 	AlarmCodeMaximumLength  = 64
-	EventIDMaximumLength    = 64
+	GUIDMaximumLength       = 64
```
What do you think if we ask Devdroid on the maximun length for this field ?

CedricPerotto replied:
We only check length on alarm objects... Either we check it everywhere or we never check it (I would personalty remove this validation).

Nas84 replied:
fine to remove this validation, it could cause us some trouble

Nas84 reviewed (commented):
Looks good , just few questions: 
- should we raise the maximum length on guid (#SeagullV2) ?
- i am not sure did we test the case where the guid is not provided and the eventId is ?

CedricPerotto committed:
Remove guid length validation for alarms

CedricPerotto committed:
RC version

CedricPerotto commented:
>     * should we raise the maximum length on guid (#SeagullV2) ?
just dropped it
>     * i am not sure did we test the case where the guid is not provided and the eventId is ?
see tests in data/types/base_test.go (and I just updated some system tests with local run).
